sbt_common/messaging/src/test/scala/uk/ac/wellcome/messaging/test/fixtures/Messaging.scala

```diff
@@ -63,7 +63,8 @@ trait Messaging
   def messageReaderLocalFlags(bucket: Bucket, queue: Queue) =
     Map(
       "aws.message.s3.bucketName" -> bucket.name,
-      "aws.message.sqs.queue.url" -> queue.url
+      "aws.message.sqs.queue.url" -> queue.url,
+      "aws.message.sqs.waitTime" -> "1",
```

Comment:
This is the other big change in this patch, and the one that substantially reduces the log size – we get significantly fewer timeouts from our elasticmq container, and thus less log spam.